src/codegen.cpp

```diff
@@ -1313,9 +1314,8 @@ static jl_method_instance_t *jl_get_unspecialized(jl_method_instance_t *method)
 
 // this compiles li and emits fptr
 extern "C"
-jl_generic_fptr_t jl_generate_fptr(jl_method_instance_t *li, void *_F, size_t world)
+jl_generic_fptr_t jl_generate_fptr(jl_method_instance_t *li, const char *F, size_t world)
```

Comment:
yeah, we just kind of pick whichever one is appropriate given whichever other variables needed to also appear. I'm not sure it's worth too much effort to try to do a sweep through and make them that much more consistent. I'm planning on eliminating these names anyways and switching just to using the fptr field directly. This PR is just part of a transitional state to slowly decouple and remove that global state.